Gruntfile.js

```diff
@@ -78,6 +78,15 @@ module.exports = function(grunt) {
           summary: true
         }
       },
+      consoleLoggingOptions: {
+        src: 'test/fixtures/pivotal/src/**/*.js',
+        options: {
+          specs: 'test/fixtures/pivotal/spec/*Spec.js',
+          helpers: 'test/fixtures/pivotal/spec/*Helper.js',
+          display: 'short',
+          showLogging: true
```

Comment:
I see what you mean. Using a testing tool to test itself is tricky business.

Using the jasmine node runner for the core tests and spawning subprocesses to run the grunt fixtures in phantomjs would have allowed for a more robust inspection of the library's behavior. That's out of scope for this PR though :no_good:.